host/webhooks/webhooks.go

```diff
@@ -0,0 +1,252 @@
+package webhooks
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"sync"
+	"time"
+)
+
+type (
+	Webhook struct {
```

Comment:
Add ID so users can reference it for the `[DELETE]` endpoint

Reply:
I missed this comment earlier so I ended up adding a GetWebhook method, which i use to look up the webhook that I send to DeleteWebhook. Would it be preferable to use ID directly or is my way ok? GetWebhook adds a sql lookup though.